### ISSUE TITLE
`check_model` and `plot.see_check_*` functions

### ISSUE THREAD

**Issue by mattansb** [easystats/performance#448]
Maybe it's just me, but the method for `check_model` seems really hard to expand / add to currently.

Would it not be easier to have `check_model` be a simple wrapper around the various `plot.see_check_*` functions, like this:

```R
check_model <- function(model, ...) {
  plot(check_normality(model)) + 
    plot(check_homogeneity(model)) + 
    plot(check_collinearity(model)) + 
    plot(check_outliers(model)) + 
    patchwork::plot_layout(ncol = 2)
}
```

With the plotting methods for each `check_*` living peacefully in see?

---

(If it is just me, then I need someone to implement `check_model.htest` 😒)

**Comment by strengejacke** [comment]
Yeah, I think is due to that this function has grown over time, and the plot methods for each single functions were partly added later... We could refactor that function, to become easier to maintain.

**Comment by bwiernik** [comment]
Yes, it's exactly what Daniel says. We should move these to methods for the specific check functions and also use visualization_recipe().
